programs/marinade-finance/src/state/validator_system.rs

```diff
@@ -73,6 +73,39 @@ impl ValidatorRecord {
     }
 }
 
+#[derive(Debug, Clone, AnchorSerialize, AnchorDeserialize)]
+pub struct ValidatorList {}
+
+impl Discriminator for ValidatorList {
+    const DISCRIMINATOR: [u8; 8] = *b"validatr";
+}
+
+impl AccountDeserialize for ValidatorList {
+    fn try_deserialize_unchecked(buf: &mut &[u8]) -> Result<Self> {
+        *buf = &buf[8..];
+        Ok(Self {})
+    }
+
+    fn try_deserialize(buf: &mut &[u8]) -> Result<Self> {
+        if buf.len() < 8 {
+            return err!(MarinadeError::InvalidValidatorListDiscriminator);
+        }
+        if buf[0..8] != Self::DISCRIMINATOR {
+            return err!(MarinadeError::InvalidValidatorListDiscriminator);
+        }
+        *buf = &buf[8..];
+        Ok(Self {})
+    }
+}
+
+impl AccountSerialize for ValidatorList {}
```

Comment:
@aankor just to confirm, when you do this:
`impl AccountSerialize for ValidatorList {}`
Then Anchor does not try to write the account data at the end of the instruction, correct?
If that's so can you please add a comment saying:
"By implementing an empty AccountSerialize, Anchor will not try to write the account data at the end of the instruction. The account data is written explicitly in the code"

Reply:
Added comment